libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectUtils.kt

```diff
@@ -0,0 +1,47 @@
+@file:JvmName("RoadObjectUtils")
+
+package com.mapbox.navigation.base.trip.model.roadobject
+
+import androidx.annotation.IntRange
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.core.constants.Constants
+import com.mapbox.geojson.LineString
+import com.mapbox.navigation.base.utils.ifNonNull
+
+private const val PRECISION_RANGE_START = 5L
+private const val PRECISION_RANGE_END = 6L
+
+/**
+ * Returns a [LineString] geometry of the road object.
+ *
+ * @param directionsRoute the original route that this object is a part of.
+ * @param precision precision of the polyline encoding.
+ * Possible values are [Constants.PRECISION_5] and [Constants.PRECISION_6].
+ * See [RouteOptions.Builder.geometries].
+ */
+fun RoadObjectGeometry.toLineString(
+    directionsRoute: DirectionsRoute,
+    @IntRange(from = PRECISION_RANGE_START, to = PRECISION_RANGE_END) precision: Int
+): LineString? {
+    if (precision !in PRECISION_RANGE_START..PRECISION_RANGE_END) {
+        throw IllegalArgumentException(
+            "Precision must be in $PRECISION_RANGE_START..$PRECISION_RANGE_END"
+        )
+    }
```

Comment:
```suggestion
    require(precision in PRECISION_RANGE_START..PRECISION_RANGE_END) {
        "Precision must be in $PRECISION_RANGE_START..$PRECISION_RANGE_END"
    }
```

Reply:
both expressions say the same:
`"Precision must be in [5..6]"` is the same as 
`"The precision values for route geometry can either be 5 or 6"`

but I prefer the first one because we are consistent with ` @IntRange` annotation. We handle valid values as a `range`. And if one day we will decide to expand the range, we will just change `PRECISION_RANGE_START / PRECISION_RANGE_END`